### ISSUE TITLE
File conversion not working properly when replacing Bolt TiVo with Edge Tivo

### ISSUE THREAD

**Issue by nleibson** [mackworth/cTiVo#459]
I had been using cTivo successfully to convert recordings from my TiVo Bolt to my iMac for over two years.  I recently replaced my Bolt with a new TiVo Edge. Now every file I convert is corrupted.  The converted video is pixilated and the audio goes on and off.  The source video on the TiVo is playing fine on the TiVo.  I have tried changing to all of the various video formats in the format dropdown and they all produce the same corrupted file.  I even added the decrypted TiVo format to the format dropdown and tried that and that also produced a corrupted file.

Any help would be appreciated.
Norm

**Comment by mackworth** [comment]
Sorry to hear it. The Decrypted result suggests there’s not much we can do, as that’s about as unmodified a file as we can do. 

But there’s a couple of things to check. First, see if your signal strength is good in the TiVo troubleshooting menus. (For example, could be a loose cable connector). Any viewing trouble at all on the TiVo?  Do you have the TiVo phone app to try?

Second, if you want to increase logging to Detail and post the result, I can check if there’s any indications there. 

Third, any differences between channels? check in Edit channels and confirm that you don’t have any unusual settings. Any difference between TS channels and non? Any difference between higher/lower channels?

**Comment by nleibson** [comment]
Thanks so much for the quick response and the detailed suggestions.  
1) I checked the channel signal strength on the channels I copied and they are all at 100% and no viewing problems on the TiVo and I do not have the TiVo Phone app. 
[com.cTiVo.cTiVo 2021-06-05--19-58-11-053.log.zip](https://github.com/mackworth/cTiVo/files/6604613/com.cTiVo.cTiVo.2021-06-05--19-58-11-053.log.zip)
2) I increased the logging details and I have attached the log file. 3)  All channels are copying the same both high and low and I checked in Edit Channels and saw nothing unusual and no difference between TS channels and non.
